randrctl/ctl.py

```diff
@@ -193,29 +194,80 @@ def _config_safe(self, config: dict, section: str, property: str):
 
     def _configs(self):
         """
-        :return: generator of config files in defined home directories if exist
+        :return: dictionary of config files in defined home directories if exist
         """
+        # Empty container to hold the configs
+        configs = {}
         for homedir in self.homes:
-            config = os.path.join(homedir, self.config_name)
-            if os.path.isfile(config):
-                yield config
+            # Configs will be either user's (prefered) or system-wide (other)
+            is_favourite = 'preferred' if homedir == self.preferred_home else 'other'
```

Comment:
I think `is_favourite` is not exactly is favourite but generally denotes the group in which config falls in. Should be renamed to something that represents its meaning better. If you do not have better ideas, what about `config_group`?